playwright-specflow-xunit/Steps/StepDefinitions.cs

```diff
@@ -0,0 +1,88 @@
+﻿using Microsoft.Playwright;
+using Newtonsoft.Json;
+
+namespace  SpecFlowPlaywrightXUnitExample.Steps
+{
+    [Binding]
+    public class PlaywrightTestParallel
+    {
+        private IBrowser? _browser;
+        private Dictionary<string, object>? _capabilities = new Dictionary<string, object>();
+        private string gdpUrl = "";
+        private string currentUrl;
+
+        [BeforeScenario]
+        private async void CreateCapabilities()
+        {
+            string user, accessKey;
+            user = Environment.GetEnvironmentVariable("LT_USERNAME");
+            accessKey = Environment.GetEnvironmentVariable("LT_ACCESS_KEY");
+            
+            Dictionary<string, string> ltOptions = new Dictionary<string, string>();
+
+            ltOptions.Add("name", "Playwright Test");
+            ltOptions.Add("build", "Playwright C-Sharp XUnit tests");
+            ltOptions.Add("platform", "Windows 10");
+            ltOptions.Add("user", user);
+            ltOptions.Add("accessKey", accessKey);
+            
+            _capabilities.Add("browserVersion", "latest");
+            _capabilities.Add("LT:Options", ltOptions);
+        }
+        
+        [AfterScenario]
+        private async void CloseDriver()
+        {
+            await _browser.CloseAsync();
+        }
+
+        [Given(@"Set (.*) as a capability")]
+        public void CreateBrowser(string browserType)
+        {
+            _capabilities.Add("browserName", browserType);
+            string capabilitiesJson = JsonConvert.SerializeObject(_capabilities);
+            gdpUrl = "wss://cdp.lambdatest.com/playwright?capabilities=" + Uri.EscapeDataString(capabilitiesJson);
+        }
+    
+        [Then(@"Go to Bing")]
+        public async Task SearchWithBing()
+        {
+            using var playwright = await Playwright.CreateAsync();
+            _browser = await playwright.Chromium.ConnectAsync(gdpUrl);

```

Comment:
@alisengul-lambdatest , can we create teh test in BeforeSceranrio rather than creating the test in the function itself? We can pass the page object to the individual functions. Here, its just creating the URL in before scenario function.

Reply:
@alisengul-lambdatest , cant we create the test in before scenario via the params? As we can get the config required for a specific scenario in the beforeScenario block too.

Reply:
As far as I know we cannot give params to the BeforeMethod from feature file. It is running before scenario started. But maybe we can I don't know actually. @ankur-lt